x-pack/plugins/ml/public/application/components/help_menu/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
```

Comment:
@jgowdyelastic would it be better to use `useMlKibana` for access to the `docLinks` service here?

Reply:
for consistency with our other components, yes `useMlKibana ` would be better. 
but `chrome` and `docLinks` are available in either.